apps/notifications/src/app/note-list/dataviews-overrides.scss

```diff
@@ -1,10 +1,13 @@
 @import '@wordpress/base-styles/colors';
+@import "@wordpress/base-styles/variables";
```

Comment:
Nit: Single quote for consistency.

Reply:
Done by https://github.com/Automattic/wp-calypso/pull/105593/commits/1998a9683e765889e0d633f4e7c724c9a5fcd614.